starter_code/src/movies.js

```diff
@@ -1,20 +1,132 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
 
+function turnHoursToMinutes(movies) {
+    return movies.map(function(movie) {
+        movie = Object.assign({}, movie);
+        var duration = movie.duration.split(' ').reduce(function(acc, time) {
+          if (time.indexOf('h') !== -1) {
+              return Number(time.replace('h', '')) * 60 + acc;
+          } else {
+              return Number(time.replace('min', '')) + acc;
+          }
+        }, 0);
+        movie.duration = duration;
+        return movie;
+    });
```

Comment:
Está bastante genial, pero así por ser quejica, habéis pensado dividirlo en dos funciones?